com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/util/ReportGenerator.java

```diff
@@ -149,14 +147,21 @@ private void parseFromPublisher(Publisher p) {
     private void parseArchiveBuilder(ArchiveBuilder archiveBuilder) {
         List<ArchiveEntry> includedEntries = archiveBuilder.getIncludedEntries();
         List<ArchiveEntry> excludedEntries = archiveBuilder.getExcludedEntries();
+        int padding = archiveBuilder.getResourcePadding();
 
         for (ArchiveEntry archiveEntry : includedEntries) {
-            long compressedSize = archiveEntry.isCompressed() ? archiveEntry.getCompressedSize() : archiveEntry.getSize();
+            long compressedSize = 0;
+            long size = 0;
+            if (!archiveEntry.isDuplicatedDataBlob()) {
+                compressedSize = archiveEntry.isCompressed() ? archiveEntry.getCompressedSize() : archiveEntry.getSize();
+                compressedSize = ((compressedSize + padding - 1) / padding) * padding;
```

Comment:
This formula adds alignment paddings into the file size.  
With alignment padding of 4, it's 0–3 bytes.